README.md

```diff
@@ -1,24 +1,28 @@
-# Homebrew
-Features, usage and installation instructions are [summarised on the homepage](http://brew.sh).
+# **Homebrew**
+**Features, usage and installation instructions are [summarised on the homepage](http://brew.sh).**
 
 ## What Packages Are Available?
-1. Type `brew search` for a list.
-2. Or visit [braumeister.org](http://braumeister.org) to browse packages online.
-3. Or use `brew search --desc <keyword>` to browse packages from the command line.
+There are 3 ways you can find packages:
++ Use `brew search` in the command line for a list of packages.
++ Use `brew search --desc <keyword>` to browse packages from the command line.
++ Alternatively, you can visit [braumeister.org](http://braumeister.org) to browse packages online.
 
 ## More Documentation
-`brew help`, `man brew` or check [our documentation](https://github.com/Homebrew/brew/tree/master/share/doc/homebrew#readme).
+Use `brew help` or `man brew` in the command line. Additionally, check [our online documentation](https://github.com/Homebrew/brew/tree/master/share/doc/homebrew#readme).
```

Comment:
Again, “on the command line” instead of “in”. The “online” is really redundant as it should be pretty clear from the fact that this is rendered as a link (and seen by users that read this on GitHub).